x/ecocredit/base/client/tx.go

```diff
@@ -44,13 +45,21 @@ Parameters:
 - issuers:    	       comma separated (no spaces) list of issuer account addresses
 - credit-type-abbrev:  the abbreviation of a credit type
 - metadata:            arbitrary data attached to the credit class info
-- fee:                 fee to pay for the creation of the credit class`,
+
+Flags:
+
+- class-fee: the fee that the class creator will pay to create the credit class. It must be >= the
+    required credit_class_fee param. We include the fee explicitly here so that the
+    class creator explicitly acknowledges paying this fee and is not surprised to learn that they
+    paid a big fee and didn't know beforehand.
+
```

Comment:
would be nice to add that the fee is not req'd when the param is empty. 

also, not sure if this suggestion is formatted correctly so may want to just edit directly 😄 
```suggestion
- class-fee: the fee that the class creator will pay to create the credit class. It must be >= the
    required credit_class_fee param. If the credit_class_fee param is empty, no fee is required. 
    We include the fee explicitly here so that the class creator explicitly acknowledges paying 
    this fee and is not surprised to learn that the they paid a big fee and didn't know beforehand.

```